dspace/config/dspace.cfg

```diff
@@ -28,8 +28,8 @@ dspace.hostname = localhost
 # DSpace base host URL.  Include port number etc.
 dspace.baseUrl = http://localhost:8080
 
-# The user interface you will be using for DSpace. Common usage is either xmlui or jspui
-dspace.ui = xmlui
+# The user interface you will be using for DSpace.
+dspace.ui = dspace-spring-rest
```

Comment:
Was this change on purpose? I'm not against this conceptually, but we may wish to consider (eventually) renaming the config, as obviously the REST app is no longer a "ui".  Perhaps it should be `dspace.path` or even `dspace.rest.url` eventually (as presumably, we should support having REST running on a separate server from the Angular UI)